zebrad/src/commands/start.rs

```diff
@@ -60,7 +60,7 @@ impl StartCmd {
 
         let mut syncer = sync::Syncer::new(peer_set, state, verifier);
 
-        syncer.run().await
+        syncer.sync().await
```

Comment:
👍

Reply:
(The real reason I did this was so that the `#[instrument]`-generated span would have a nice filterable name).